src/Symfony/Component/DependencyInjection/Loader/Configurator/PrototypeConfigurator.php

```diff
@@ -40,7 +40,6 @@ class PrototypeConfigurator extends AbstractServiceConfigurator
     private $loader;
     private $resource;
     private $exclude;
-    private $allowParent;
```

Comment:
this should be kept (same below), this is a false positive: the property is used by a trait

Reply:
But that makes the trait dependent on the class using it? Which is in turn dependent on the trait it's using?